packages/ui-app/src/Params/types.ts

```diff
@@ -13,7 +13,7 @@ export type RawParam$ValueArray = Array<RawParam$Value | Array<RawParam$Value>>;
 
 export type RawParam$Values = RawParam$Value | RawParam$ValueArray;
 
-export type RawParam = {
+export type RawParam = TypeDef & {
```

Comment:
i tried creating a commit https://github.com/polkadot-js/apps/commit/ab948b660dcc1545545f6ca1902dcf8e5a86c6ed in separate branch 'luke-417-display-params-for-storage-queries-3-typedef' to try using using a `typeDef` property internally, but I get errors when i run it because `param` doesn't have a `typeDef` property.

when i check the changes with `yarn run check` it doesn't show any errors so i thought the changes would also resolve the CI errors, but when i tried querying storage in the UI at http://localhost:3000/#/chainstate it gives error `Uncaught TypeError: Cannot read property 'info' of undefined` since `param` doesn't have a `typeDef` property, and it compiles from TypeScript to JS as `const paramInfo = param.typeDef.info;`